guava-tests/test/com/google/common/collect/SynchronizedTableTest.java

```diff
@@ -0,0 +1,329 @@
+            assertTrue(Thread.holdsLock(mutex));
+            return super.columnMap();
+        }
+
+        @Override
+        public boolean contains(Object rowKey, Object columnKey) {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.contains(rowKey, columnKey);
+        }
+
+        @Override
+        public boolean containsColumn(Object columnKey) {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.containsColumn(columnKey);
+        }
+
+        @Override
+        public boolean containsRow(Object rowKey) {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.containsRow(rowKey);
+        }
+
+        @Override
+        public boolean containsValue(Object value) {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.containsValue(value);
+        }
+
+        @Override
+        public V get(Object rowKey, Object columnKey) {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.get(rowKey, columnKey);
+        }
+
+        @Override
+        public boolean isEmpty() {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.isEmpty();
+        }
+
+        @Override
+        public V put(R rowKey, C columnKey, V value) {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.put(rowKey, columnKey, value);
+        }
+
+        @Override
+        public void putAll(Table<? extends R, ? extends C, ? extends V> table) {
+            assertTrue(Thread.holdsLock(mutex));
+            super.putAll(table);
+        }
+
+        @Override
+        public V remove(Object rowKey, Object columnKey) {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.remove(rowKey, columnKey);
+        }
+
+        @Override
+        public Map<C, V> row(R rowKey) {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.row(rowKey);
+        }
+
+        @Override
+        public Set<R> rowKeySet() {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.rowKeySet();
+        }
+
+        @Override
+        public Map<R, Map<C, V>> rowMap() {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.rowMap();
+        }
+
+        @Override
+        public int size() {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.size();
+        }
+
+        @Override
+        public Collection<V> values() {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.values();
+        }
+
+        @Override
+        public boolean equals(Object obj) {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.equals(obj);
+        }
+
+        @Override
+        public int hashCode() {
+            assertTrue(Thread.holdsLock(mutex));
+            return super.hashCode();
+        }
+
+        private static long serialVersionUID = 0;
+    }
+
+  /*
```

Comment:
I think it's this comment that should be..indented +2?  The leading / of the comment should be at the same level as the method signatures.